package.json

```diff
@@ -96,7 +104,7 @@
     "start-js": "react-scripts-ts start",
     "start": "npm-run-all -p watch-css start-js",
     "build": "yarn build-css && react-scripts-ts build && ncp build dist",
-    "test": "react-scripts-ts test --env=jsdom",
```

Comment:
Why did you replace this with the jest command? Doesn't react-scripts-ts use jest for testing?

Reply:
There seems to be some problem with the jest config of react-scripts-ts (https://github.com/facebook/jest/issues/6393). It doesn't really matter since both commands just run jest in watch mode.

Reply:
Not exactly, react-scripts-ts runs ts-jest not jest

Reply:
I am using ts-jest as a preset in the jest config